package.json

```diff
@@ -148,7 +148,7 @@
   "bundlesize": [
     {
       "path": "./dist/styled-components.min.js",
-      "threshold": "14kB"
+      "threshold": "14.25kB"
```

Comment:
Adding the `minify` code takes the minified bundle from `13.96kB` to `14.25kB`. It might be possible to refactor some of it to reduce size.